### PR TITLE
fix(core): replace with GLPI-Agent

### PR DESCRIPTION
**Pull request by stonebuzz**
fix(core): replace with GLPI-Agent

### CODE REVIEW

**Review by g-bougard — CHANGES_REQUESTED** [review]
Indeed GLPI-Agent 2.3.20 & GLPI-Agent 2.3.21 don't exist.
You should probably change text to tell to use GLPI-Agent or FIA with the given versions.

**Comment by stonebuzz** [comment]
> Indeed GLPI-Agent 2.3.20 & GLPI-Agent 2.3.21 don't exist. You should probably change text to tell to use GLPI-Agent or FIA with the given versions.

done =)

File: inc/deployaction.class.php [hunk 1]
```diff
@@ -439,7 +439,7 @@ public function displayAjaxValues($config, $request_data, $rand, $mode)
                     ];
             Dropdown::showNumber('logLineLimit', $options);
             echo "&nbsp;<span class='red'><i>";
-            echo __('Fusioninventory-Agent 2.3.20 or higher mandatory');
+            echo __('GLPI-gent or Fusioninventory-Agent >= 2.3.20 mandatory');
```
Comment (cedric-anne) [inline inc/deployaction.class.php:442]:
```suggestion
            echo __('GLPI-Agent or Fusioninventory-Agent >= 2.3.20 mandatory', 'glpiinventory');
```

File: inc/deploycheck.class.php [hunk 2]
```diff
@@ -351,36 +351,36 @@ public function getLabelsAndTypes($check_type, $mandatory = false)
             case "winkeyMissing":
                 $values['path_label']         = __("Path to the key", 'glpiinventory') . $mandatory_mark;
                 $values['value_label']     = false;
-                $values['path_comment']    = __('Example of registry key') . ': HKEY_LOCAL_MACHINE\SOFTWARE\Fusioninventory-Agent\\';
-                $values['warning_message'] = sprintf(__('Fusioninventory-Agent %1s or higher recommended'), '2.3.20');
+                $values['path_comment']    = __('Example of registry key') . ': HKEY_LOCAL_MACHINE\SOFTWARE\GLPI-Agent\\';
```
Comment (cedric-anne) [inline inc/deploycheck.class.php:354]:
```suggestion
                $values['path_comment']    = __('Example of registry key', 'glpiinventory') . ': HKEY_LOCAL_MACHINE\SOFTWARE\GLPI-Agent\\';
```

File: inc/deploycheck.class.php [hunk 3]
```diff
@@ -351,36 +351,36 @@ public function getLabelsAndTypes($check_type, $mandatory = false)
...
+                $values['warning_message'] = sprintf(__('GLPI-Agent or Fusioninventory-Agent >= %1s recommended'), '2.3.20');
```
Comment (cedric-anne) [inline inc/deploycheck.class.php:355]:
```suggestion
                $values['warning_message'] = sprintf(__('GLPI-Agent or Fusioninventory-Agent >= %1s recommended', 'glpiinventory'), '2.3.20');
```

File: inc/deploycheck.class.php [hunk 4]
```diff
@@ -351,36 +351,36 @@ public function getLabelsAndTypes($check_type, $mandatory = false)
...
                 break;
 
             case "winvalueExists":
             case "winvalueMissing":
                 $values['path_label']      = __("Path to the value", 'glpiinventory') . $mandatory_mark;
                 $values['value_label']     = false;
-                $values['path_comment']    = __('Example of registry value') . ': HKEY_LOCAL_MACHINE\SOFTWARE\Fusioninventory-Agent\server';
-                $values['warning_message'] = sprintf(__('Fusioninventory-Agent %1s or higher mandatory'), '2.3.20');
+                $values['path_comment']    = __('Example of registry value') . ': HKEY_LOCAL_MACHINE\SOFTWARE\GLPI-Agent\server';
```
Comment (cedric-anne) [inline inc/deploycheck.class.php:362]:
```suggestion
                $values['path_comment']    = __('Example of registry value', 'glpiinventory') . ': HKEY_LOCAL_MACHINE\SOFTWARE\GLPI-Agent\server';
```

**Review by cedric-anne — CHANGES_REQUESTED** [review]
Lots of messages are not using the `glpiinventory` translation domain, it should be fixed.

**Review by g-bougard — APPROVED** [review]

File: inc/deployaction.class.php [hunk 5]
```diff
@@ -439,7 +439,7 @@ public function displayAjaxValues($config, $request_data, $rand, $mode)
                     ];
             Dropdown::showNumber('logLineLimit', $options);
             echo "&nbsp;<span class='red'><i>";
-            echo __('Fusioninventory-Agent 2.3.20 or higher mandatory');
+            echo __('GLPI-Agent or Fusioninventory-Agent >= 2.3.20 mandatory', 'glpiinventory');
```
Comment (cedric-anne) [inline inc/deployaction.class.php:442]:
```suggestion
            echo sprintf(__('GLPI-Agent or Fusioninventory-Agent >= %1s mandatory', 'glpiinventory'), '2.3.20');
```

**Pull request merged**